src/server_manager/ui_components/outline-server-view.html

```diff
@@ -544,6 +562,10 @@ <h3>[[localize('server-help-access-key-title')]]</h3>
         isAccessKeyPortEditable: {type: Boolean, value: false},
         serverCreationDate: String,
         serverLocation: String,
+        accessKeyDataLimit: {type: Object, value: undefined},
+        isAccessKeyDataLimitEnabled: {type: Boolean},
+        canSetAccessKeyDataLimit: {type: Boolean, value: false},
```

Comment:
Could you add a comment explaining this property and `dataLimitsAvailabilityDate`?

Reply:
Done.

Reply:
I had added it to `outline-server-settings`, which also declares them. Added here now.